plugins/janus_videoroom.c

```diff
@@ -2997,10 +2998,13 @@ static int janus_videoroom_access_room(json_t *root, gboolean check_modify, gboo
 		/* signed tokens bypass pin validation */
 		json_t *token = json_object_get(root, "token");
 		if(token) {
-			char room_descriptor[26];
+			char room_descriptor[100];
```

Comment:
Was this the actual cause of the issue? A string that was too small, and so was only containing a partial representation of the required token causing comparisons and checks to fail?

Reply:
This affects on checking rooms with id in UUID format, the description have to be: `room=3edb0cdd-7173-481d-82f0-5eb31937d20f` but when it has only 26 characters it's looks like: `room=3edb0cdd-7173-481d-82` and when we do strcmp we have not equal result.